x-pack/test_serverless/functional/test_suites/common/security/navigation/avatar_menu.ts

```diff
@@ -11,7 +11,9 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
   const svlCommonPage = getPageObject('svlCommonPage');
   const svlCommonNavigation = getService('svlCommonNavigation');
 
-  describe('Avatar menu', function () {
+  // Requires SSL and security enabled on ES Serverless.
+  // https://github.com/elastic/kibana/issues/162625
+  describe.skip('Avatar menu', function () {
```

Comment:
I could be missing something here but this test requires authentication, which requires security to be enabled, which in turn requires SSL (#162625).